frontend/packages/core/src/Input/tests/toggle-button-group.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { ToggleButton } from "@material-ui/lab";
+import { shallow } from "enzyme";
+
+import ToggleButtonGroup from "../toggle-button-group";
+
+// Note interaction tests will happen when there is a parent component that
+// can maintain state and select a default
+describe("ToggleButtonGroup", () => {
+  let multipleFalseWrapper;
+  let multipleTrueWrapper;
+  beforeEach(() => {
+    multipleFalseWrapper = shallow(
+      <ToggleButtonGroup value="foo" onChange={() => {}} size="large">
+        <ToggleButton value="foo">foo</ToggleButton>
+        <ToggleButton value="Ingress">Ingress</ToggleButton>
+        <ToggleButton value="Egress">Egress</ToggleButton>
+      </ToggleButtonGroup>
+    );
+
+    multipleTrueWrapper = shallow(
+      <ToggleButtonGroup
+        multiple
+        value={["foo", "Ingress"]}
+        onChange={() => {}}
+        orientation="vertical"
+      >
+        <ToggleButton value="foo">foo</ToggleButton>
+        <ToggleButton value="Ingress">Ingress</ToggleButton>
+      </ToggleButtonGroup>
+    );
+  });
+  const searchStringForComponent = "Styled(WithStyles(ForwardRef(ToggleButton)))";
+
+  describe("rendering", () => {
+    it("renders", () => {
+      expect(multipleFalseWrapper.find(ToggleButtonGroup)).toBeDefined();
+    });
+
+    it("renders with 3 toggles", () => {
+      expect(multipleFalseWrapper.find(ToggleButton)).toHaveLength(3);
+    });
+
+    it("renders with 2 toggles", () => {
+      expect(multipleTrueWrapper.find(ToggleButton)).toHaveLength(2);
+    });
```

Comment:
These tests are identical so I'm not sure we're going to glean much from it. We should likely be testing that if there is no length that it doesn't render or renders 'x' etc.